test/upload_test.sh

```diff
@@ -67,20 +76,21 @@ stop_network_conditions() {
             "sudo dnctl -q flush"
         )
     else
-        # TODO implement commands for linux
-        return 0
+        stop_condition_commands=(
+            "sudo tc qdisc del dev lo root"
+        )
     fi
 
-    for command in "${set_condition_commands[@]}"; do
-        echo $root_password | sudo -S bash -c "$command"
+    for command in "${stop_condition_commands[@]}"; do
+        bash -c "$command"
```

Comment:
Shouldn't this use `sudo`? Also, couldn't you use `eval` rather than starting another shell?